AllReadyApp/Web-App/AllReady.UnitTest/Services/Routing/GoogleOptimizeRouteServiceShould.cs

```diff
@@ -7,33 +7,129 @@
 using Moq;
 using AllReady.Services.Routing;
 using Microsoft.Extensions.Options;
+using System.Net.Http;
+using System.IO;
+using System.Text;
 
 namespace AllReady.UnitTest.Services.Routing
 {
     public class GoogleOptimizeRouteServiceShould
     {
+        private readonly OptimizeRouteCriteria _optimiseRouteCriteria;
+        private readonly Guid _requestId1 = new Guid("1d582827-af66-4eea-b405-9c1feaaee68d");
+        private readonly Guid _requestId2 = new Guid("aa14b9ba-8ed4-428f-9413-f69d2b832795");
+
+
+
+        public GoogleOptimizeRouteServiceShould()
+        {
+            _optimiseRouteCriteria = new OptimizeRouteCriteria(
+                "24 Sussex Drive Ottawa ON",
+                "200 Sussex Drive Ottawa ON",
+                new List<OptimizeRouteWaypoint>()
+                    {
+                        new OptimizeRouteWaypoint(100.0, 100.0, _requestId1),
+                        new OptimizeRouteWaypoint(200.0, 200.0, _requestId2)
+                    }
+                );
+        }
+        [Fact]
+        public async Task ReturnsOptimisedRoutes()
```

Comment:
@mgmccarthy  Thanks for all the helpful and thorough feedback.
- You are right the unit tests should not have external dependencies. However I believe serialising and then deseralising an object for the unit test setup can lead to stale tests because the test is now dependent on making sure that the serialising model is correct. I opted to place the google api response as a string in the source file to keep the test good.